app/assets/stylesheets/navigation.scss

```diff
@@ -0,0 +1,89 @@
+@import "colors";
```

Comment:
not so interesting question, but can / should we move this file to webpack too?

Reply:
patternfly uses variables from this file, so I don't quite sure if moving it works well.